### PR TITLE
Move to the 'black' code style.

### PR DESCRIPTION
**Pull request by srevinsaju**
`flake8` is old and cumbersome, its painful, and a waste of time. The [PSF](https://en.wikipedia.org/wiki/Python_Software_Foundation) has created, and recommends new users to use the `black` code style, which is simple and ubobstrusive. 

The benefits of using `black` is that,
* it has an autolinter,
* It keeps a balance between readability and line limits
* The code looks exactly the same, despite how many times your run the linter on it. 

We can even attach a GitHub Action to automatically lint new user's pull requests, but I guess, I will leave that for discussion.
I have seen some projects adding `.git/hooks/pre-commit`, which runs the black linter before every commit. So all commits are neater than ever.

PS: the change from `'` to `"` is to signify i18n, as convention. `black` strictly enforces the same. 

Let me know your thoughts. I have migrated most of my projects to `black`.

### CODE REVIEW

**Comment by Uzay-G** [comment]
I'll look and review this after v1, don't want to introduce changes like this right before :)

File: README.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ Logo design by [Roy Quilor](https://www.quilor.com/) is licensed under [CC BY-NC
 
 # Archivy
 
+[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
```
Comment (Uzay-G) [inline README.md:7]: I don't think we need this to be put so prominently on the README, could you remove it?

**Comment by Uzay-G** [comment]
@srevinsaju could you please resolve the conflicts? Then if everything works well I'll merge this in :)

**Comment by srevinsaju** [comment]
Done.

**Comment by Uzay-G** [comment]
Looks good :)

**Pull request merged**